gapidapk/deviceinfo.go

```diff
@@ -104,30 +104,52 @@ func fetchDeviceInfo(ctx context.Context, d adb.Device) error {
 	// Close any previous runs of the apk
 	apk.Stop(ctx)
 
-	driver, err := d.GraphicsDriver(ctx)
-	if err != nil {
-		return err
-	}
-
 	var cleanup app.Cleanup
 
-	// Set up device info service to use prerelease driver.
-	nextCleanup, err := adb.SetupPrereleaseDriver(ctx, d, apk.InstalledPackage)
-	cleanup = cleanup.Then(nextCleanup)
+	packages := []string{}
+	driver, err := d.GraphicsDriver(ctx)
 	if err != nil {
-		cleanup.Invoke(ctx)
-		return err
+		// If there's an error, keep going to attempt to use GPU profiling
+		// libraries in system image.
+		log.W(ctx, "Failed to query developer driver: %v, assuming no developer driver found.", err)
+	}
+	if driver.Package != "" {
+		log.I(ctx, "Use GPU profiling libraries from developer driver package.")
```

Comment:
nit: "Using..."

And, since we're already logging, we should log the package name:
```suggestion
		log.I(ctx, "Using GPU profiling libraries from developer driver package: %v", driver.Package)
```

(same below)